defaults/main.yml

```diff
@@ -52,6 +53,7 @@ openwisp2_sentry:
 openwisp2_default_cert_validity: 1825
 openwisp2_default_ca_validity: 3650
 openwisp2_default_organization_id: 78f74fdd-67c6-4064-97e1-ce761da30745
+openwisp2_notifications_delete_old_notifications: 90
```

Comment:
don't forget to add these two new variables in the [Role Variables docs](https://github.com/openwisp/ansible-openwisp2#role-variables)